### ISSUE TITLE
Test coverage

### ISSUE THREAD

**Issue by sjakobi** [haskell-unordered-containers/unordered-containers#270]
I just ran `stack test --coverage` on https://github.com/haskell-unordered-containers/unordered-containers/commit/964edca40d415c7f0b70c99fda2a4d9dc0949b9f. Here's the summary of the combined coverage report for all testsuites:

![image](https://user-images.githubusercontent.com/3664523/85168007-0a75ef00-b26a-11ea-9ea1-acb7246052ba.png)

Not terrible, but could be better. Maybe there's even some dead code we could get rid of?

**Comment by treeowl** [comment]
'Twould be nice to have more detail on what's not covered.

**Comment by sjakobi** [comment]
Yeah. It's hard to verbally summarize though. Maybe you can just generate the report yourself and take a look.

Sorry for the vague issue description. I think we should probably try to aim for >95% or so.

**Comment by treeowl** [comment]
Please provide the information above in an alternative accessible format by editing the issue text.
